### DISCUSSION TITLE
lock entity

### DISCUSSION THREAD

**Discussion by ghost** [anhaehne/hhnl.HomeAssistantNet#29] [Q&A]
Hi, 
Is it possible to call a lock-entity via CS4HA?

Best regards

**Comment by anhaehne** [comment]
Lock entities should be available in the `Entities` class where you can get the state of the lock as string.

To lock and unlock you can simply call a the corresponding service like this: https://github.com/anhaehne/hhnl.HomeAssistantNet/wiki/3.-Calling-services#other-services

I will also create a lock entity in an upcoming release.

**Reply by anhaehne** [reply]
The version 11.3 contains a new lock entity.
